textBasedAdventure.js

```diff
@@ -1,5 +1,457 @@
+        endVents();
+    } else{
+        console.clear();
+        console.log("The aliens find you suspicious because you are not dancing with them. They stop dancing and become aggressively violent towards you.")
+        player.health -= 100;
+        console.log(player);
+        console.log("You have been probed a thousand times and didn't make it out of AREA 51 ... GAME OVER!! TRY AGAIN")
+    }
+} // END leftVent()
+
+const rightVent = () => {
+    console.log("You crawl down the right vent and it starts to shake. The vent collapses and you fall down out of the vent. ")
+    damage(player);
+    if(player.health <= 0){
+        console.log("The fall from the vents disabled you to move any further. You lay there until you meet your demise.. You didn't make it out of AREA 51 alive... GAME OVER!! TRY AGAIN");
+    } else{
+        console.log(player);
+        let userInput = readline.question("You are in critical condition, but still survived that fall from the vent. Struggling but managing to get up, you see a door in front of you. Open it? ");
+        while(userInput.toLowerCase() !== 'open' && userInput.toLowerCase() !== 'open door' && userInput.toLowerCase() !== 'open the door' && userInput.toLowerCase() !== 'open it'){
+            userInput = readline.question("How much more can you take?? The only move is to go forward. Open the door! ");
+        }
+        if (userInput.toLowerCase() === "open" || userInput.toLowerCase() === "open door" || userInput.toLowerCase() === "open the door" || userInput.toLowerCase() === "open it"){
+            console.clear();
+            endVents();
+        }
+    }
+}  // END rightVent()
+
+const endVents = () => {
+    console.log("It seems that you are now located on the upper floor of the underground base. This room has one lever that is attached to the left wall and a door on the right.")
+    let userInput = readline.question("Will you pull the lever or go through the door on the right? ");
+    while(userInput.toLowerCase() !== 'pull lever' && userInput.toLowerCase() !== 'pull the lever' && userInput.toLowerCase() !== 'lever' && userInput.toLowerCase() !== 'go through door' && userInput.toLowerCase() !== 'door' && userInput.toLowerCase() !== 'go through the door' && userInput.toLowerCase() !== 'go through the door on the right' && userInput.toLowerCase() !== 'through the door' && userInput.toLowerCase() !== 'right' && userInput.toLowerCase() !== 'right door'){
+        userInput = readline.question("Pull the lever or go through the door? ");
+    }
+    if (userInput.toLowerCase() === 'pull lever' || userInput.toLowerCase() === 'pull the lever' || userInput.toLowerCase() === 'lever'){
+        console.clear();
+        secretLeverDoor();
+    } else {
+        console.clear();
+        secretRight();
+    }
+} // END endVents()
+
+let password = ["THE", "UPSIDE", "DOWN"];   // PASSWORD array NEEDED at end of game
```

Comment:
I don’t really understand why the password is an array – do you have a special plan for it? Why not using a string?